### PR TITLE
Friendship v2

### PR DESCRIPTION
**Pull request by layzrd**
add mutual friendship

### CODE REVIEW

File: app/models/friendship.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class Friendship < ApplicationRecord
+  attribute :status, :string, default: 'requested'
   belongs_to :user
 end
```
Comment (lekegitrepo) [inline app/models/friendship.rb:4]: - **_when friendship is confirmed, an additional row with reversed user-friend pair is created_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Good job so far :+1: you have done a great job. However, you need to implement the following :point_down: requirements before this project can be approved.

- **_Enable linter._**

- **_Only the user's posts and their friends' posts should be visible to the user._**

**_Fix these and request another review._**

#### _Reviewed by Temitayo Adeleke_

File: app/models/friendship.rb (hunk 1 above)
Comment (layzrd) [reply app/models/friendship.rb:4]: I didn't design to be like that. When the friend request is confirmed, the status would be changed.

**Comment by layzrd** [comment]
I enabled the linter but, GitHub activates the linter when I create a pull request upon the main repo. If you can help me to initiate here(comparing with the forked repo branch), I appreciate that.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED 🍾 

Good job with the changes. Everything looks okay now. 

I am giving this a pass. All the best with the next tasks @abredi 🥇 

#### Christian Otieno (TSE)

**Pull request merged**